### DISCUSSION TITLE
Does 'firstDay# not work with resourceTimeline

### DISCUSSION THREAD

**Discussion by JenniferSchoen** [vkurko/calendar#392] [Q&A]
I've tried changing my calendar to a different type and then it works but it won't change anything for the resourceTimeline. 

My Code (svelte):
```
[...]
let options = {
        view: 'resourceTimelineMonth',
        url: [...],
        eventSources: [...],
        resources: [...],

        hiddenDays: [6, 0],
        firstDay: 2,
        duration: {days: 14},

[...]
    };
```
Everything else, aside from the firstDay, works.

**Comment by vkurko** [comment]
[firstDay](https://github.com/vkurko/calendar?tab=readme-ov-file#firstday) option makes sense when displaying weeks. But in month mode, days are always displayed from the first day of the month. `firstDay` does not affect the monthly view.
